backend/routeHandler.js

```diff
@@ -74,6 +74,50 @@ module.exports = function configureRoutes(options) {
     res.send(JSON.stringify(query));
   });
 
+  options.router.post('/getQueryByName', async (req, res) => {
+    try {
+    // Checking templates
+      const getQueryParams = {
+        sourceKey: req.body.sourceKey,
+        type: "template"
+      };
+
+      let response = (await options.getRestClient(req).graphQuery.getQueries(getQueryParams));
+
+      let queries = response.body.filter((q) => {
+        return q.name.toLowerCase() === req.body.name;
+      });
+
+      // Checking statics
+      getQueryParams.type = "static"
+
+      response = (await options.getRestClient(req).graphQuery.getQueries(getQueryParams));
+
+      queries = queries.concat(response.body.filter((q) => {
+        return q.name === req.body.name;
```

Comment:
@andre91a I think you miss the `.toLowerCase()` on both sides of the condition (same comments as before)

Reply:
@fantandrea I forgot the lowercase here

Reply:
fixing it